src/platform/intel/ace/lib/pm_runtime.c

```diff
@@ -9,12 +9,56 @@
 #include <sof_versions.h>
 #include <stdint.h>
 
+#include <zephyr/pm/policy.h>
+
 /* 76cc9773-440c-4df9-95a8-72defe7796fc */
 DECLARE_SOF_UUID("power", power_uuid, 0x76cc9773, 0x440c, 0x4df9,
 		 0x95, 0xa8, 0x72, 0xde, 0xfe, 0x77, 0x96, 0xfc);
 
 DECLARE_TR_CTX(power_tr, SOF_UUID(power_uuid), LOG_LEVEL_INFO);
 
+#ifdef CONFIG_PM_POLICY_CUSTOM
+const struct pm_state_info *pm_policy_next_state(uint8_t cpu, int32_t ticks)
+{
+	uint8_t num_cpu_states;
```

Comment:
I see that `pm_state_cpu_get_all()` returns `uint8_t` but `unsigned int` would work here too